engine/redis_v2/engine_test.go

```diff
@@ -394,7 +394,7 @@ func TestEngine_Peek_SecondaryStorage(t *testing.T) {
 		ID:         "",
 		Body:       body,
 		TTL:        120,
-		Delay:      15,
+		Delay:      45,
```

Comment:
It turned out to be that the job's delay in previous test case is not long enough, that it got pumped in redis before actually peeking the job. And therefore didn't reach the error code.
I fixed the test job delay to ensure that the job is still stored in DB when peeking job.

Reply:
ok, make sense